imagersite/imager_images/tests.py

```diff
@@ -0,0 +1,84 @@
+from django.test import TestCase, Client
+from imager_images.models import Photo
+from imager_images.models import Album
+import factory
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.urls import reverse
+import os
+from imagersite.settings import MEDIA_ROOT
+from django.contrib.auth.models import User
+
+
+class UserFactory(factory.django.DjangoModelFactory):
+    """Setting up users for tests."""
+    class Meta:
+        model = User
+    username = factory.Sequence(lambda n: "user{}".format(n))
+    email = factory.Sequence(
+        lambda n: "user{}@example.com".format(n)
+    )
+
+
+class PhotoFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = Photo
+    title = factory.Sequence(lambda n: "photo{}".format(n))
+    image = SimpleUploadedFile(
+        name="testing.png",
+        content=open(MEDIA_ROOT + '/test/testing.png', 'rb').read(),
+        content_type="image/png"
+    )
+
+
+class AlbumFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = Album
+    title = factory.Sequence(lambda n: "album{}".format(n))
+
+
+class ProfileTests(TestCase):
+
+    def setUp(self):
+        user = UserFactory.create()
+        user.set_password('caaarlos')
+        user.save()
+        self.user = user
+        photos = [PhotoFactory.create(profile=user.profile) for i in range(20)]
+        album = AlbumFactory.build()
+        album.profile = user.profile
+        album.save()
+        for photo in photos:
+            album.photos.add(photo)
+        album.cover_photo = photos[0]
+        self.photos = photos
+        self.album = album
+        self.client = Client()
+
+    def tearDown(self):
+        to_delete = os.path.join(MEDIA_ROOT, 'photos', 'testing*.png')
+        os.system('rm -rf ' + to_delete)
+
+    def test_some_route_lists_images(self):
```

Comment:
Some route? Is the profile supposed to list images? Or is it supposed to list data about the user's profile?